.env.example

```
@@ -1,15 +1,6 @@
-
-# For production with real Ollama API
-# DEBUG=false
-# USE_DEMO_RESPONSES=false
+# For remote: OLM_API_ENDPOINT=http://your-server:8000
 # OLM_API_ENDPOINT=http://localhost:8000
-# OLLAMA_MODEL=qwen3:0.6b
-
-# LLM Processing Settings
-USE_TWO_STAGE_LLM=true
\ No newline at end of file
+OLLAMA_MODEL=qwen3:0.6b
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)